### PR TITLE
Made ipa file optional for the hockey action

### PR DESCRIPTION
**Pull request by KrauseFx**
Fixes https://github.com/fastlane/fastlane/issues/1327

### CODE REVIEW

File: lib/fastlane/actions/hockey.rb [hunk 1]
```diff
@@ -19,6 +19,10 @@ def self.run(options)
         if options[:dsym]
           dsym_filename = options[:dsym]
         else
+          if options[:ipa].to_s.length == 0
+            UI.user_error!("You have to provide an ipa file")
```
Comment (asfalcone) [inline lib/fastlane/actions/hockey.rb]: Should this be "You must provide an ipa or dsym"?

Comment (KrauseFx) [inline lib/fastlane/actions/hockey.rb]: in this case it must be an ipa file, only if the `upload_dsym_only` is set, a dSYM is enough. The most common use case is to upload a binary though, so the error message should be fine.

**Comment by KrauseFx** [comment]
Seems like the build is failing, looking into this

**Comment by KrauseFx** [comment]
Updated :+1:

**Comment by asfalcone** [comment]
LGTM!

**Pull request merged**